### PR TITLE
Task4_with_PageObject 

### PR DESCRIPTION
**Pull request by rahim1sa**

### CODE REVIEW

**Comment by ElenaQQQ** [comment]
Great work! I may only suggest to delete ApptTest file.

**Review by YaninaProkharava — COMMENTED** [review]
Good decision to create a method for Wait!

**Comment by rahim1sa** [comment]
> 

Done!

**Comment by rahim1sa** [comment]
> Great work! I may only suggest to delete ApptTest file.

Done!

File: src/main/java/org/example/pageobject/pages/listedproducts/ListedByPriceRangePage.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.example.pageobject.BasePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import java.util.List;
+
+public class ListedByPriceRangePage extends BasePage {
+    @FindBy(xpath = "//*[@id=\"s-refinements\"]/div[12]/ul/li")
+            public List<WebElement> priceRangeCount;
```
Comment (juliaHrabovska) [inline src/main/java/org/example/pageobject/pages/listedproducts/ListedByPriceRangePage.java:12]: It is better to make these variables private. And format your code. Use CTRL + ALT + L

Comment (rahim1sa) [reply src/main/java/org/example/pageobject/pages/listedproducts/ListedByPriceRangePage.java:12]: Done!

**Pull request merged**